test/e2e/clusterinfo/constants.go

```diff
@@ -1,10 +1,11 @@
 package clusterinfo
 
 const (
-	MachineOSIDLabel = "machine.openshift.io/os-id"
-	MachineSetLabel  = "machine.openshift.io/cluster-api-machineset"
-	MachineRoleLabel = "machine.openshift.io/cluster-api-machine-role"
-	MachineTypeLabel = "machine.openshift.io/cluster-api-machine-type"
+	KubernetesIOClusterLabelPrefix = "kubernetes-io-cluster-"
```

Comment:
I dont see this label in the OCP docs https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-gcp.html
If its required shouldn't it be on the example machinesets?